code/modules/antagonists/revolution/revolution.dm

```diff
@@ -303,6 +303,18 @@
 		new_target.target = M
 		new_target.update_explanation_text()
 		objectives += new_target
+
+	var/untracked_protagonists = SSstation.protagonists
+	for(var/datum/objective/mutiny/O in objectives)
+		untracked_protagonists -= O.target
+
+	for(var/datum/mind/M in untracked_protagonists)
```

Comment:
better var names, please. also, use "as anything" here if `untracked_protagonists` is only minds

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names
and
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#avoid-unnecessary-type-checks-and-obscuring-nulls-in-lists